content/posts/2024/08/12/goodbye-to-legacy-game-recaps-and-leaderboards/index.md

```diff
@@ -0,0 +1,29 @@
++++
+title = 'Changes to how we handle older games'
+date = 2024-08-09T14:51:52-07:00
+draft = true
+authors = ["saman", "orta"]
+tags = ["games", "changelog", "sunset"]
++++
+
+Hey folks, this week we've had a think about how we can ensure some of the long-term systems running in Puzzmo keep running smoothly. We concluded that there are two things we should probably do to keep site speed up, and keep costs down.
+
+1. **We are removing the stats displays on games completed before April 1st, 2024.**
+
+    ![A diagram showing three sidebars: 1) the sidebar before sunsetting legacy results for games completed before April 1st, 2) the sidebar after sunsetting for games completed before April 1st, and 3) the sidebar for games completed after April 1st](illustration.png)
+
+    The version of Puzzmo which launched, and the Puzzmo we all use today may visually look very similar, but under-the-hood things are quite different. There's a _long_ technical [write-up here](https://blog.puzzmo.com/posts/2024/07/16/augmentations/) but TLDR: we have new tools which powers things like custom club leaderboards, _[un-announced features in Pile-Up Poker]_ and Puzzle Remixes.
+
+    We've been keeping the older systems around to make sure the migration back in April was seamless, but have been finding these older system have been slowing down work on the 2 new games we have in active development. So, we're starting to remove these pieces.
+
+    This only affects when you look at games completed prior to April 1st, 2024, it will not affect your statistics/streaks etc.
```

Comment:
"This only affects When you look" is a bit awkward. Maybe "you will only notice the difference when you look..." or "this change only affects games..."